device/thunder_ripple_sdk/src/processors/thunder_device_info.rs

```diff
@@ -1116,16 +1116,6 @@ impl ThunderDeviceInfoRequestProcessor {
 
         // If timezone or offset is None or empty
         if let Some(tz) = Self::get_timezone_and_offset(&state).await {
-            let cloned_state = state.clone();
```

Comment:
We get the Timezone and set it in Ripple context here. This sets up the initial context until the point the user changes the timezone. Are we removing this because it is adding that log?

Reply:
Same question.

Reply:
@sdk, I removed it because i understood like, that cause's the warning log. I'll revert this change.